src/sentry/api/event_search.py

```diff
@@ -694,18 +694,9 @@ def to_list(value):
             ]
         ],
     },
-    "p75": {
-        "result_type": "duration",
-        "aggregations": [["quantile(0.75)(duration)", "", "p75"]],
-    },
-    "p95": {
-        "result_type": "duration",
-        "aggregations": [["quantile(0.95)(duration)", "", "p95"]],
-    },
-    "p99": {
-        "result_type": "duration",
-        "aggregations": [["quantile(0.99)(duration)", "", "p99"]],
-    },
+    "p75": {"result_type": "duration", "aggregations": [["quantile(0.75)(duration)", "", "p75"]]},
+    "p95": {"result_type": "duration", "aggregations": [["quantile(0.95)(duration)", "", "p95"]]},
+    "p99": {"result_type": "duration", "aggregations": [["quantile(0.99)(duration)", "", "p99"]]},
```

Comment:
Weird, why'd these lines change? Seems too long to me

Reply:
black doing black things.